projects/lookup/src/modules/autocomplete/types/autocomplete-search-async-args.ts

```diff
@@ -0,0 +1,42 @@
+import { SkyAutocompleteSearchAsyncResult } from '@skyux/lookup';
+import { Observable } from 'rxjs';
+import { AutocompleteSearchAsyncResultDisplayType } from './autocomplete-search-async-result-display-type';
+
+/**
+ * Arguments passed when an asynchronous search is executed from the
+ * autocomplete or lookup component.
+ */
+export interface SkyAutocompleteSearchAsyncArgs {
+
+  /**
+   * The search text entered by the user.
+   */
+  searchText: string;
+
+  /**
+   * How the search results will be displayed. Use this value to determine
+   * how many results to return.
+   */
+  displayType: AutocompleteSearchAsyncResultDisplayType;
+
+  /**
+   * The offset index of the first result to return. When search is executed
+   * as a result of an infinite scroll event, for example, offset will be set
+   * to the number of items already displayed.
+   */
+  offset: number;
+
+
+  /**
+   * A continuation token which can be set and then will be passed back with any future searches.
+   * This is helpful for applications which utilize a token instead of an offset when fetching data.
+   */
+  continuationData?: unknown;
```

Comment:
Should this just be called `continuationToken`

Reply:
The comment probably just needs tweaked as I finish things up. I went with `continuationData` based on a conversation I had with Paul before he went on PTO. The more generic term makes sure that it is open ended for any use of the property.